src/simulation/m_time_steppers.fpp

```diff
@@ -941,8 +941,8 @@ contains
 
         elseif (bubbles_lagrange) then
 
-            call s_populate_variables_buffers(q_prim_vf, pb_ts(1)%sf, mv_ts(1)%sf, bc_type)
-            call s_compute_bubble_EL_dynamics(q_prim_vf, stage)
+            call s_populate_variables_buffers(q_prim_vf, pb_ts(1)%sf, mv_ts(1)%sf, bc_type, bc_bound)
+            call s_compute_bubble_EL_dynamics(q_cons_ts(1)%vf, q_prim_vf, t_step, rhs_vf, stage)
```

Comment:
is this correct? looks like a breaking change to the subroutine call? or maybe this is from a merge conflict you fixed